environments/icds-cas/public.yml

```diff
@@ -154,7 +154,7 @@ localsettings:
   COUCH_CACHE_VIEWS: True
   COUCH_STALE_QUERY: 'update_after'
   DAYS_TO_KEEP_DEVICE_LOGS: 45
-  DEPLOY_MACHINE_NAME: "{{ ansible_hostname }}"
+  DEPLOY_MACHINE_NAME: "{{ alt_hostname|default(ansible_hostname) }}"
```

Comment:
Maybe I missed, But I do not see where `alt_hostname` is defined.

Reply:
It's in the CSV inventory plugin: https://github.com/dimagi/commcare-cloud/pull/2077/files#diff-a4ffb19691318917608e6abbff958e45R164

Reply:
A bit sneaky putting it there - we could also just define a column in the inventory file and add it there but it would basically duplicate the first column

Reply:
Thanks.